app/validators/language_code_validator.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class LanguageCodeValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    record.errors.add(attribute, :language_code_invalid) unless valid?(value)
```

Comment:
Could you add the message for this error in `/api/config/locales/en.yml`?

Reply:
I always forget it:) Added!